app/src/main/java/com/example/cinemapp/data/Movie.kt

```diff
@@ -5,17 +5,17 @@ import java.util.Date
 
 data class Movie(
     @SerializedName("adult") val adult: Boolean? = null,
-    @SerializedName("backdrop_path") val backdropPath: String? = null,
+//    @SerializedName("backdrop_path") val backdropPath: String? = null,
```

Comment:
Commented-out fields should be deleted, and brought back only when they are needed again.